src/software/export/main_exportAnimatedCamera.cpp

```diff
@@ -157,18 +238,71 @@ int aliceVision_main(int argc, char** argv)
       const std::string dstImage = (undistortedImagesFolderPath / (std::to_string(view.getIntrinsicId()) + "_" + imagePathStem + "." + image::EImageFileType_enumToString(outputFileType))).string();
       const camera::IntrinsicBase * cam = iterIntrinsic->second.get();
 
-      image::readImage(view.getImagePath(), image, image::EImageColorSpace::LINEAR);
+      image::readImage(view.getImagePath(), image, image::EImageColorSpace::NO_CONVERSION);
+      oiio::ParamValueList metadata = image::readImageMetadata(view.getImagePath());
 
       if(cam->isValid() && cam->hasDistortion())
       {
         // undistort the image and save it
         camera::UndistortImage(image, cam, image_ud, image::FBLACK, true); // correct principal point
-        image::writeImage(dstImage, image_ud, image::EImageColorSpace::LINEAR);
+        if(outputFileType == image::EImageFileType::EXR)
+        {
+            // build a ROI
+            std::string key = std::to_string(view.getIntrinsicId());
+            oiio::ROI roi;
+            if(roiForIntrinsic.find(key) == roiForIntrinsic.end())
+            {
+                const camera::IntrinsicBase& intrinsic = *(cam);
```

Comment:
It would be good to put the computation of the ROI of an intrinsic in a function.